### PR TITLE
Document how to suggest lints for flutter_lints

### PR DESCRIPTION
**Pull request by goderbauer**
## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

File: packages/flutter_lints/README.md [hunk 1]
```diff
@@ -58,7 +58,21 @@ linter:
 # https://dart.dev/guides/language/analysis-options
 ```
 
+## Adding new lints
+
+Please file a [lint proposal] issue to suggest that an existing lint rule should
```
Comment (mit-mit) [inline packages/flutter_lints/README.md:63]: I think you need `[lint proposal][]` for that link to work?

**Review by mit-mit — COMMENTED** [review]
LGTM, just one comment

File: packages/flutter_lints/README.md (hunk 1 above)
Comment (goderbauer) [reply packages/flutter_lints/README.md:63]: The style without the `[]` also works: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

I also verified that the links further up in the readme using the same style link correctly, see https://pub.dev/packages/flutter_lints/versions/2.0.0.

File: packages/flutter_lints/README.md [hunk 2]
```diff
@@ -58,7 +58,21 @@ linter:
...
+be added to this packages. The benefits and risks of adding a lint should be
```
Comment (pq) [inline packages/flutter_lints/README.md:64]: => this package.

File: packages/flutter_lints/README.md [hunk 3]
```diff
@@ -58,7 +58,21 @@ linter:
...
+Adding a lint to the package may create new warnings for existing users and is
+therefore considered to be a breaking change, which will require a major version
+bump. To keep churn low, lints are not added one-by-one, but in one batch
+following a review of all accumulated suggestions since the previous review.
+
 [Flutter]: https://flutter.dev
 [dart analyzer]: https://dart.dev/guides/language/analysis-options
 [Dart-enabled IDEs]: https://dart.dev/tools#ides-and-editors
 [package:lints]: https://pub.dev/packages/lints
+[lint proposal]: https://github.com/dart-lang/lints/issues/new/choose
```
Comment (pq) [inline packages/flutter_lints/README.md:78]: Consider going a step past the chooser?

https://github.com/dart-lang/lints/issues/new?&labels=type-lint&template=lint-propoposal.md

**Review by pq — APPROVED** [review]

**Pull request merged**